binlog-stream/src/main/scala/io/laserdisc/mysql/binlog/compaction/package.scala

```diff
@@ -25,12 +25,6 @@ package object compaction {
               case None => acc.remove(emId)
             }
           }
-        // TODO: use this code, once Scala 2.12 support ends, or  updateWith back ported to 2.12A
-//          acc.updateWith(evt.pk) {
-//            case Some(latest) =>
-//              mkNewEvent(evt).andThen(finalizeNewEvent).run(latest)
-//            case None => Some(evt)
-//          }
```

Comment:
I tried this, it puts events in reverse order - I think the existing impl is fine